src/app/components/new-board/new-board.component.ts

```diff
@@ -423,158 +443,6 @@ export class NewBoardComponent implements OnInit {
         this.AddPdfRenderComponent.addPdfRenderToolBox(uid, event.target.files[0], this.reader);
       });
 
-      // Add code snippet
```

Comment:
why are we removing this? I am seeing few unexpected behaviour :-
1. Image plugin isn't working
2. Code snippet plugin is providing the option for choosing the URL (If I am not wrong that should appear for image plugin)
3. Graph plugin isn't behaving as expected for json input file.

Are these the the bugs or only happening with this PR?

Reply:
1. Okay we were planning to remove the image plugin and code snippet plugin. Corresponding changes in html code should be made.
2. Code snippet plugin was redundant before so is being removed.
3. file upload plugin can be used for image plugin and pdf upload functionality.
4. Graph plugin feature didn't know it was breaking. i have to confirm with @goliakshay357 .